Newsletters: add 263 (2023-08-09)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin Core PR Review Club summary @LarryRuane 
- [x] Bitcoin Core 27746 @adamjonas 
- [x] Core Lightning 6376 @murchandamus

LarryRuane commented:
I've run into a problem due to the review club meeting now spanning two days, related to the meeting log links. (We link to meeting log lines from the questions-and-answers part of the review club write-up.) I created an [issue](https://github.com/bitcoin-core-review-club/website/issues/710) on the review club repo.

If they can't fix this in the next day or 2, would it be acceptable to move the review club section to next week?

harding commented:
@LarryRuane postponing is fine any month but especially faultless this month as we wouldn't normally have one of our monthly sections next week (it's a five-Wednesday month and we've never had anything regular scheduled for middle weeks).

glozow commented:
> I've run into a problem due to the review club meeting now spanning two days, related to the meeting log links.

Fixed, sorry about that!

LarryRuane commented:
Since @glozow fixed the problem so quickly (thanks!), I went ahead and pushed the review club commit, but feel free to back it out and include it next week.
(Just did a force-push to remove a trailing space to fix the CI failure.)

bitschmidty commented on _posts/en/newsletters/2023-08-09-newsletter.md:
```diff
@@ -0,0 +1,245 @@
+  that would be charged in addition to the usual fee for a successful
+  payment.  As an HTLC propagated from spender Alice to forwarding node
+  Bob, Alice would pay Bob a small forward commitment fee; this would
+  correspond to Bob's cost for processing the HTLC (such as bandwidth).
```
"this would correspond to Bob's cost for processing the HTLC"

But later we outline that the fee could also be used to compensate future participants along the route which, to me, makes this sentence confusing or incorrect (unless we include his forwarding fees as part of the processing of the HTLC, in addition to the bandwidth example)

bitschmidty commented on _posts/en/newsletters/2023-08-09-newsletter.md:
```diff
@@ -0,0 +1,245 @@
+  - *Full RBF by default:* the [first thread][todd rbf] and [pull
+    request][bitcoin core #28132] proposes making [full RBF][topic rbf]
+    the default setting in a future version of Bitcoin Core.  Currently,
+    Bitcoin Core by default will only relay and accept into its mempool
```
Maybe

```suggestion
    Bitcoin Core, by default, will only relay and accept into its mempool
```

LarryRuane commented on _posts/en/newsletters/2023-08-02-newsletter.md:
```diff
@@ -46,6 +46,93 @@ Bitcoin infrastructure software.
     find the thread interesting for its discussion of cryptographic
     techniques. {% assign timestamp="5:07" %}
 
+## Bitcoin Core PR Review Club
```
> @LarryRuane this went into last weeks newsletter (262) instead of this weeks (263)

Oh, I guess we're not doing time travel yet ... sorry about that, fixed.

LarryRuane commented on _posts/en/newsletters/2023-08-09-newsletter.md:
```diff
@@ -0,0 +1,333 @@
+      business that currently accepts unconfirmed onchain transactions
+      as final payment.
+
+  - *Removing specific limits on `OP_RETURN` outputs:* the [second
```
Possibly link to https://developer.bitcoin.org/devguide/transactions.html#null-data for background?

harding replied:
That section has an error I'm aware of (and haven't gotten around to fixing), so gonna skip that for now.

LarryRuane commented on _posts/en/newsletters/2023-08-09-newsletter.md:
```diff
@@ -0,0 +1,333 @@
+
+      The pull request would default to allowing any number of
+      `OP_RETURN` outputs and any amount of data in an `OP_RETURN`
+      output as long as the transactions otherwise obeyed Bitcoin Core's
+      relay policy (e.g., had a total transaction size less than 100,000
```
```suggestion
      output as long as the transactions obey Bitcoin Core's
      other relay policies (e.g., had a total transaction size of less than 100,000
```

harding commented on _posts/en/newsletters/2023-08-09-newsletter.md:
```diff
@@ -0,0 +1,333 @@
+  onion encryption so that each forwarding node only knows what peer it
+  received the message from and what peer should next receive the
+  message.  Message payloads are also encrypted so that only the
+  ultimate receiver can read it.  Unlike forwarded HTLCs, which are
```
Keeping as-is.  I think this should be singular.

harding commented:
Added security announcement for the libbitcoin thing.  Made edits or left comments on all feedback (thanks @LarryRuane @bitschmidty !) and added lede and releases and topic entries.  Reviewed excellent contributions from @LarryRuane @adamjonas @murchandamus (thanks!), with extra thanks to @glozow for the upstream fix.  Big, exciting issue this week, thanks all!

bitschmidty commented:
Pushed a few small fixups, squashed and merged, thank you @harding @murchandamus @LarryRuane @adamjonas @glozow for a great newsletter! 🚀

murchandamus commented on _includes/templates/compatibility/rbf.md:
```diff
@@ -112,10 +112,10 @@ block chain space used, this is the most efficient form of fee bumping.
     Allows sending of BIP125 opt-in-RBF transactions in the interface.
   {% when "false" %}{:.feature-no}
-  - **Does not signal BIP125 replacability when sending transactions**<br>
+  - **Does not signal BIP125 repleacability when sending transactions**<br>
```
This introduced a typo, fixing it here:  #1237
